docs/Help.md

```diff
@@ -120,3 +120,7 @@ Nightly ignores version checking and it will not attempt to update on its own. B
 
 ##### Issues
 Since nightly is a testing release, it may be a little unstable. If you find any bugs, make sure to report an issue on [GitHub](https://github.com/Chatterino/chatterino2/issues/new/choose). Before opening an issue make sure you're on latest nightly (to check it, go to Settings > About and compare commit hash) and that there's no duplicate issue already open.
+
+### I see white borders on full screen
+This is a Windows issue it can be mitigated by turning on Compatibility mode for Chatterino. Right click `chatterino.exe` -> Properties -> Compatibility -> Enable "Run this program in compatibility mode for:" -> Select Windows 8 in the drop-down menu. 
+![Like this](images/help/borders.png)
```

Comment:
The text shown between the square brackets is alternative text for users who cannot load the image. "Like this" does not accurately describe the image.